css/style.css

```diff
@@ -0,0 +1,146 @@
+/* GENERAL STYLING */
+
+.crimson-font-color{
+  color: #540002;
+}
+
+.link, .link:visited{
+  color: #666F75;
+  opacity: 0.6;
+}
+
+.link:hover, .link:focus, .link:active {
+  text-decoration: none;
+  opacity: 1;
+  color: #636C72;
+}
+
+.item {
+  padding-bottom: 40px;
+}
+
+.page-head {
+  padding: 50px 0;
+}
+
+/* STYLE CONTAINERS */
+
+.cover .container{
+  padding-top: 200px; 
+}
+
+body, html {
+  height: 100%;
+}
+
+#about{
+  margin-bottom: 150px;
+  height: auto;
+  background: white;
+  color: #540002;
+}
+
+/* BUTTON PROPERTIES */
+
+.btn-info{
+  background: #74FF11;
+  color: #540002 !important;
+  border-color: #74FF11;
+}
+
+.btn-info:hover{
+  background: #64D612;
+  color: #540002 !important;
+  border-color: #64D612;
+}
+
+.btn{
+  padding: 10px 24px !important;
+  font-size: 18px;
+  margin: 3px 0 0 8px;
+  width: 130px;
+}
+
+/* NAVBAR COMPONENTS */
+
+.navbar{
+  background: transparent;
+}
+
+.navbar-toggler{
+  margin-top: 15px;
+  font-size: 25px;
+  border: none;
+}
+
+.navbar-brand{
+  font-size: 30px;
+}
+
+/* FOOTER ELEMENTS */
+
+.designer_name, .designer_name:visited{
+  text-decoration: none;
+  color: #767676;
+}
+
+.designer_name:hover, .designer_name:active, .designer_name:focus{
+  text-decoration: none;
```

Comment:
Be careful with resetting bootstrap's styles of the `:focus` pseudo selector. It's usually set as an accessibility feature (for those with handicaps that need to use screen readers and such).